app/templates/suppliers/details.html

```diff
@@ -5,19 +5,30 @@
 {% endblock %}
 
 {% block breadcrumb %}
-  {%
-    with items = [
+  {% with items = [
       {
-        "link": "/",
-        "label": "Digital Marketplace"
+        "text": "Digital Marketplace",
+        "href": "/"
       },
       {
-        "link": url_for(".dashboard"),
-        "label": "Your account"
-      }
+        "text": "Your account",
+        "href": url_for('.dashboard')
+      },
     ]
   %}
-    {% include "toolkit/breadcrumb.html" %}
+    {% if currently_applying_to %}
+      {{ govukBreadcrumbs({"items": items + [
+        {
+          "text": "Continue with your application",
+          "href": url_for(".framework_dashboard", framework_slug=framework.slug)
```

Comment:
I think the link needs to be the same as the one on line 216, this url gives me a 404.

Also, could we make the text `Apply to { framework.name }` to match the breadcrumb for the page we're coming from?